modules/router-store/src/init_router_store.ts

```diff
@@ -0,0 +1,19 @@
+import { ENVIRONMENT_INITIALIZER, inject, Provider } from '@angular/core';
+import { StoreRouterConnectingService } from './store_router_connecting.service';
+
+/**
+ * Initializer for the router-store that ensures the initialization logic is run alongside environment
+ * initialization.
+ */
+export const _initRouterStore: Provider[] = [
+  {
+    provide: ENVIRONMENT_INITIALIZER,
+    multi: true,
+    useFactory() {
+      return () => {
+        inject(StoreRouterConnectingService);
+      };
+    },
+  },
+  StoreRouterConnectingService,
+];
```

Comment:
We can delete this file and move `ENVIRONMENT_INITIALIZER` to `provideRouterStore`.